### PR TITLE
Update CHANGELOG with 2.0.x backport change entries

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1: Update backport changelog](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -275,6 +275,81 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - `@cumulus/message/Queue.getQueueName`
   - `@cumulus/message/Queue.getQueueNameByUrl`
 
+## v2.0.2+ Backport releases
```
Comment (markdboyd) [inline CHANGELOG.md:278]: I guess this is helpful content, but what was the motivation for adding it? I would have been fine just labeling the other 2.0.x releases as "backports", which you did

File: CHANGELOG.md [hunk 2]
```diff
@@ -275,6 +275,81 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+Release v2.0.1 was the last release on the 2.0.x release series.
+
+Changes after this version on the 2.0.x release series are limited
+security/requested feature patches and will not be ported forward to future
+releases unless there is acorresponding CHANGELOG entry.
```
Comment (markdboyd) [inline CHANGELOG.md:284]:
```suggestion
releases unless there is a corresponding CHANGELOG entry.
```

File: CHANGELOG.md (hunk 1 above)
Comment (markdboyd) [reply CHANGELOG.md:278]: Regardless, since you already wrote it, might as well leave it in

File: CHANGELOG.md (hunk 2 above)
Comment (markdboyd) [reply CHANGELOG.md:284]: I really hate blocking merge of this PR on something like this, so I'm also happy if we want to just not worry about this one

File: CHANGELOG.md (hunk 1 above)
Comment (Jkovarik) [reply CHANGELOG.md:278]: I think there's confusion over what the CHANGELOG on master is, given our backport structure.     

Just adding the [BACKPORT] label doesn't necessarily give readers context as to what that means (fixes mentioned aren't promised to have been brought into master/future versions as-described), and a disclaimer/gentle nudge in the right direction for the backport fixes feels warranted.

**Review by markdboyd — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md]: It's simple enough to fix and re-run, I don't think this PR's priority is such that we have to allow mistakes in.

**Pull request merged**